rdflib/namespace/__init__.py

```diff
@@ -333,11 +333,37 @@ def __init__(self, graph):
         self.__trie = {}
         for p, n in self.namespaces():  # self.bind is not always called
             insert_trie(self.__trie, str(n))
-        self.bind("xml", XMLNS)
+
+        self.bind("brick", BRICK)
+        self.bind("csvw", CSVW)
+        self.bind("dc", DC)
+        self.bind("dcat", DCAT)
+        self.bind("dcmitype", DCMITYPE)
+        self.bind("dcterms", DCTERMS)
+        self.bind("dcam", DCAM)
+        self.bind("doap", DOAP)
+        self.bind("foaf", FOAF)
+        self.bind("odrl", ODRL2)
+        self.bind("org", ORG)
+        self.bind("owl", OWL)
+        self.bind("prof", PROF)
+        self.bind("prov", PROV)
+        self.bind("qb", QB)
         self.bind("rdf", RDF)
         self.bind("rdfs", RDFS)
+        self.bind("schema", SDO)
+        self.bind("sh", SH)
+        self.bind("skos", SKOS)
+        self.bind("sosa", SOSA)
+        self.bind("ssn", SSN)
+        self.bind("time", TIME)
+        self.bind("vann", VANN)
+        self.bind("void", VOID)
         self.bind("xsd", XSD)
 
+        # TODO: XMLNS is not a DefinedNamespace.
```

Comment:
Is there unique properties in the XMLNS? If not, then we can remove the todo comment.

Reply:
No, there are no properties for it. It's not an RDF namespace as such, just a legacy XML one. I _think_ people need it for RDF/XML but no other RDF formats care about it.

Reply:
Cool thanks, I removed the todo comment in https://github.com/RDFLib/rdflib/pull/1491/commits/ce9e2cf4003b195f15911f45f28e99c2bc59a302.